packages/meditrak-server/src/routes/importSurveyResponses/importSurveyResponses.js

```diff
@@ -312,3 +311,15 @@ function getCellContents(sheet, columnIndex, rowIndex) {
   const cell = sheet[xlsx.utils.encode_cell({ c: columnIndex, r: rowIndex })];
   return cell === undefined ? '' : cell.v; // Extract the value of the cell if it wasn't blank
 }
+
+async function getSurveyFromSheet(models, sheet) {
+  const firstSurveyResponseId = getColumnHeader(sheet, INFO_COLUMN_HEADERS.length);
+  if (!firstSurveyResponseId) {
+    throw new Error(
+      'Each tab of the import file must have at least one previously submitted survey as the first entry',
+    );
+  }
+
+  const firstSurveyResponse = await models.surveyResponse.findById(firstSurveyResponseId);
+  return models.survey.findById(firstSurveyResponse.survey_id);
+}
```

Comment:
There were some code conflicts that were not resolved when merging dev back into the epic branch so the changes in this file are mostly for cleaning it up